README.md

```diff
@@ -66,48 +65,15 @@ If you don't want to post github pull requests but a random quote from your team
 
 ### Slack configuration
 
-You should also set up the following custom emojis in Slack:
-
-- :informative_seal:
-- :angrier_seal:
-- :seal_of_approval:
-- :happyseal:
-- :halloween_informative_seal:
-- :halloween_angrier_seal:
-- :halloween_seal_of_approval:
-- :festive_season_informative_seal:
-- :festive_season_angrier_seal:
-- :festive_season_seal_of_approval:
-- :manatea:
-
-You can use the images in images/emojis that have the corresponding names.
```

Comment:
This looks worth keeping somewhere (maybe rename this section to `Slack setup`?)

Reply:
I removed it because we already have those emojis in our Slack and it should be pretty obvious that you should only use existing emojis in your messages, I think? Happy to bring it back if you think it's helpful.

Reply:
I'd keep it - if we ever lose our Slack or move workspaces or the custom emojis get cleared or whatever, it would be handy to have a reference of Slack emoji that the app is expecting, so that we can. It can sort of be [derived from the code](https://github.com/alphagov/seal/blob/40120cc461b83a8df95d80206d47f114f207aa7b/lib/slack_poster.rb#L55-L77) but then you also have to figure out all the supported 'season symbols'.

Alternatively I'd be happy if we just linked to the [seal/images/emoji](https://github.com/alphagov/seal/tree/main/images/emoji) folder and said something along the lines of "This app expects that you have imported all of the emoji from [this folder], using the name of the image as the emoji name, e.g. `angrier_seal.jpg` => `:angrier_seal:`.